### ISSUE TITLE
Glimpse runs not just on localhost

### ISSUE THREAD

**Issue by mkarlsson** [avanderhoorn/Glimpse.Prototype#120]
I get Glimpse UI bar at the bottom on staging and production as well. How do I control this? I just added the nuget package and app.UseGlimpse(); in Configure method and  services.AddGlimpse(); in ConfigureServices.

**Comment by skorunka** [comment]
Hi, try "if (env.IsDevelopment()) { .... }" in your ConfigureServices((IServiceCollection services, IHostingEnvironment env) and Configure(IApplicationBuilder app, IHostingEnvironment env) methods.

**Comment by mkarlsson** [comment]
I was under the impression it was supposed to work anyway?

**Comment by skorunka** [comment]
I don't think so. You have to control if to add Glimpe into app pipeline or not.
